data/snippets.conf

```diff
@@ -46,6 +53,7 @@ do=do\n{\n\t%cursor%\n} while (%cursor%)\n%cursor%
 switch=switch (%cursor%)%brace_open%case %cursor%:\n\t\t%cursor%\n\t\tbreak;\n\tdefault:\n\t\t%cursor%\n%brace_close%%cursor%
 
 [C++]
+main=\n\nint main(int argc, char* argv[]) {\n\t%cursor%\n\treturn 0;\n}
```

Comment:
I doubt this is very useful, generally one only has on main per program -- and for e.g. unit tests with one `main` per unit, I guess a template would be better useful with extra specificities.